lib/src/authenticate.tsx

```diff
@@ -113,9 +128,9 @@ const AuthProvider: FunctionComponent<{ children: any; config: any; }> = ({ chil
                 signOut,
                 state,
                 updateConfig
-            } }
+            }}
         >
-            { configState && children }
+            {configState && children}
```

Comment:
Let's add the `space` back.
```suggestion
            { configState && children }
```